Library/Formula/libcppa.rb

```diff
@@ -1,42 +1,82 @@
+  head "https://github.com/actor-framework/actor-framework.git"
 
-  # since upstream has rename the project to actor-framework (or libcaf in its
-  # pkgconfig file), we need to rename libcppa to libcaf in the future
+  # Since CAF does not produce development releases, we repurpose --devel to
+  # install from the 'develop' branch.
+  devel do
+    url "https://github.com/actor-framework/actor-framework.git",
+      :branch => "develop"
+    version "0.13-devel"
+  end
 
   bottle do
     cellar :any
-    sha1 "b0e9bef1983d561763e21539c9b9196d75e5a935" => :yosemite
-    sha1 "ed71bb57236d2aecf4e19e5044ca3af22969b5c5" => :mavericks
-    sha1 "8224fe20d5d4bd184a9b6c15ddd6143740ea23ca" => :mountain_lion
+    sha1 "672795a5d8b6a888752de0f4dc841ceb12c6a3b6" => :yosemite
   end
 
   depends_on "cmake" => :build
 
   needs :cxx11
 
-  option "with-opencl", "Build with OpenCL actors"
-  option "with-examples", "Build examples"
-  option "without-check", "Skip build-time tests (not recommended)"
-
-  def install
-    ENV.cxx11
+  option "with-static", "build as static and shared library"
+  option "with-static-only", "build as static library only"
+  option "with-opencl", "build with support for OpenCL actors"
+  option "with-cash", "build with cash (CAF shell)"
+  option "with-nexus", "build with nexus (central runtime inspection component)"
+  option "with-riac", "build with riac (runtime inspection and configuration)"
+  option "without-unit-tests", "skip unit tests (not recommended)"
 
-    args = %W[
-      --prefix=#{prefix}
-      --build-static
-    ]
+  # Because Homebrew does not support option values, we use one option per
+  # log-level values.
+  option "with-log-level-error", "build with log level ERROR"
+  option "with-log-level-warning", "build with log level WARNING"
+  option "with-log-level-info", "build with log level INFO"
+  option "with-log-level-debug", "build with log level DEBUG"
+  option "with-log-level-trace", "build with log level TRACE"
 
+  def install
+    args = %W[./configure --prefix=#{prefix} --no-examples]
+    args << "--build-static" if build.with? "static"
+    args << "--build-static-only" if build.with? "static-only"
     args << "--no-opencl" if build.without? "opencl"
-    args << "--no-examples" if build.without? "examples"
+    args << "--no-cash" if build.without? "cash"
+    args << "--no-nexus" if build.without? "nexus"
+    args << "--no-riac" if build.without? "riac"
+    args << "--with-log-level=ERROR" if build.with? "log-level-error"
+    args << "--with-log-level=WARNING" if build.with? "log-level-warning"
+    args << "--with-log-level=INFO" if build.with? "log-level-info"
+    args << "--with-log-level=DEBUG" if build.with? "log-level-debug"
+    args << "--with-log-level=TRACE" if build.with? "log-level-trace"
+
+    # Clear state from previous builds to allow for changing build options.
+    system "make", "distclean" if File.exist?("Makefile")
```

Comment:
The reason is that it prevents using different options. Say you start with `--build-static`, abort the current build, and remove the option on the next round. Because there exists a CMake cache build option cache, you get "locked in" on the first set of options passed to `./configure`. Hence the invocation of `distclean` to allow for changing build options. My thinking was that this gives users the least unexpected behavior.